docs/book/v3/form-creation/attributes-or-annotations.md

```diff
@@ -257,7 +257,7 @@ optionally further `options` to pass to the constructor of the filter.
 as JSON.
 
 Additionally, you can use the `priority` argument to modify the order
-of the filters in the [filter chain](../../laminas-filter/filter-chains/).
+of the filters in the [filter chain](https://docs.laminas.dev/laminas-filter/filter-chains/).
```

Comment:
Are we required to have the domain in the URL by some laminas convention? If now, I'd rather go with `/laminas-filter/filter-chains/` next time :)

Reply:
All my tests for local links would fail.

> Warning
> Using absolute paths with links is not officially supported. Relative paths are adjusted by MkDocs to ensure they are always relative to the page. Absolute paths are not modified at all. This means that your links using absolute paths might work fine in your local environment but they might break once you deploy them to your production server.

https://www.mkdocs.org/user-guide/writing-your-docs/#linking-to-pages

Reply:
Nice then :+1:

Reply:
Your objection is valid. I will add a paragraph to documentation writer's guide about this.